### PR TITLE
New error message.

### PR DESCRIPTION
**Pull request by maurineimirandazup**
Signed-off-by: maurineimirandazup <maurinei.miranda@zup.com.br>

### Description
An adjustment in the color and the error message.
A friendlier message.
### How to verify it

**Before:**

**Now**

### CODE REVIEW

**Review by GuillaumeFalourd — APPROVED** [review]
LGTM 👍🏼

File: pkg/formula/builder/make.go [hunk 1]
```diff
@@ -46,7 +47,11 @@ func (ma MakeManager) Build(info formula.BuildInfo) error {
 	cmd := exec.Command("make", "build")
 	cmd.Stderr = &stderr
 	if err := cmd.Run(); err != nil {
-		return fmt.Errorf(errMsgFmt, ErrBuildFormulaMakefile, stderr.String())
+		return errors.New(
+			fmt.Sprint(
+				prompt.Red(ErrBuildFormulaShell.Error()),
```
Comment (brunasilvazup) [inline pkg/formula/builder/make.go:52]: I think you forgot to change the variable here, and the value of `msgMakeBuildErr` too.
You can verify?

`ErrBuildFormulaShell` -> `ErrBuildFormulaMakefile`

Comment (maurineimirandazup) [reply pkg/formula/builder/make.go]: Thanks for the review. 👊🏽
Fixed.

**Review by GuillaumeFalourd — APPROVED** [review]

**Review by brunasilvazup — COMMENTED** [review]
🔝

**Review by brunasilvazup — APPROVED** [review]

**Review by lucasdittrichzup — APPROVED** [review]

**Pull request merged**